build-container/entrypoint.sh

```diff
@@ -15,6 +15,7 @@ set -x
 sudo chown -R buildbot:buildbot /src
 cd /src
 
+sed -i "/^THREADS=/s/9/$(nproc)/" ~/.config/clean-chroot-manager.conf
```

Comment:
`nproc` is indeed simpler, but it gives the logical number of cores. Then again, maybe we're okay with this?

Reply:
@kerberizer Usually the advice I heard (and the strategy I went with) is to set the number of parallel builds to logical cores (sometimes +1) to take as much advantage of any SMP as possible

Reply:
@Doridian Yup, Graysky's comment was "The typical rule is physical cores + 1", but this is indeed a classical case of "it depends". It will probably be best to test different values, but that's not a priority right now.